lib/client.js

```diff
@@ -1,21 +1,24 @@
+'use strict';
 /**
  * Module dependencies
  */
-var events        = require('events')
-  , Store         = require('./store')
-  , eos           = require('end-of-stream')
-  , mqttPacket    = require('mqtt-packet')
-  , Writable      = require('readable-stream').Writable
-  , inherits      = require('inherits')
-  , setImmediate  = global.setImmediate || function(callback) {
-      // works in node v0.8
-      process.nextTick(callback);
-    };
+/*global setImmediate:true*/
+var defaultConnectOptions, defaultId, MqttClient,
+  events = require('events'),
+  Store = require('./store'),
+  eos = require('end-of-stream'),
+  mqttPacket = require('mqtt-packet'),
+  Writable = require('readable-stream').Writable,
+  inherits = require('inherits'),
+  setImmediate = global.setImmediate || function (callback) {
+    // works in node v0.8
+    process.nextTick(callback);
+  };
```

Comment:
I actually prefer to have module dependencies at the top, and then all the rest.